docs/special-tags.md

```diff
@@ -0,0 +1,102 @@
+title: Special Tags 
+description: The following tags are treated as special in Sematext Cloud and cannot be used in tag aliases, App agent YAMLs and as custom fields in logs. They are used to correlate data across multiple Apps.
+
+Tags are sent by Sematext Agents as part of every data point and they are shown in UI as filters. 
+
+The following tags are treated as special in Sematext Cloud and cannot be used in tag aliases, App agent YAMLs and as custom fields in logs. They are used to correlate data across multiple Apps. Refer to [Special Logs Fields](./logs/special-fields/) for list of special fields in logs.
+
+## Common Tags
+
+The tags below are applicable to all metrics types:
```

Comment:
I guess wherever we show "metrics" on this page, we should adjust to "metrics/logs".